routes/v1.js

```diff
@@ -305,6 +429,11 @@ api.declare({
     return;
   }
 
+  // Ensure group membership is declared, and that schedulerId isn't conflicting
+  if (!await ensureTaskGroup(this, taskId, taskDef, res)) {
+    return;
```

Comment:
How will this cause the request to fail?  By doing an empty return here, does it cause a standard error and message to be returned?